frontend/www/js/omegaup/arena/ranking.ts

```diff
@@ -88,15 +88,72 @@ export function onRankingEvents({
 export function createChart({
   series,
   navigatorData,
+  maxPoints,
+  startTimestamp,
+  finishTimestamp,
 }: {
   series: (Highcharts.SeriesLineOptions & { rank: number })[];
   navigatorData: number[][];
-}): {
-  series: (Highcharts.SeriesLineOptions & { rank: number })[];
-  navigatorData: number[][];
-} {
-  // TODO: Implement this function in a new PR
-  return { series, navigatorData };
+  maxPoints: number;
+  startTimestamp: number;
+  finishTimestamp: number;
+}): Highcharts.Options | null {
+  if (!series.length) return null;
+
+  const scoreboardColors = [
+    '#FB3F51',
+    '#FF5D40',
+    '#FFA240',
+    '#FFC740',
+    '#59EA3A',
+    '#37DD6F',
+    '#34D0BA',
+    '#3AAACF',
+    '#8144D6',
+    '#CD35D3',
+  ];
+
+  return {
+    chart: { height: 300, spacingTop: 20 },
+
+    colors: scoreboardColors,
+
+    xAxis: {
+      ordinal: false,
+      min: startTimestamp,
+      max: finishTimestamp,
+    },
+
+    yAxis: {
+      showLastLabel: true,
+      showFirstLabel: false,
+      min: 0,
+      max: maxPoints,
+    },
+
+    plotOptions: {
+      series: {
+        animation: false,
+        lineWidth: 3,
+        states: { hover: { lineWidth: 3 } },
+        marker: { radius: 5, symbol: 'circle', lineWidth: 1 },
+      },
+    },
+
+    navigator: {
+      series: {
+        type: 'line',
+        step: true,
+        lineWidth: 3,
+        lineColor: '#333',
+        data: navigatorData,
+      },
+    },
+
+    rangeSelector: { enabled: false },
+
+    series: series,
+  } as Highcharts.Options;
```

Comment:
```suggestion
  };
```

hay que evitar el `as` excepto en lugares donde se pasa a las props de un Vue en una prueba.

Reply:
Listo